### PR TITLE
Fixes boxes overlays

### PR DESCRIPTION
**Pull request by Carbonhell**
Fucking uncommented code,atleast now it's modularized

### CODE REVIEW

**Review by yoyobatty — APPROVED** [review]

**Comment by yoyobatty** [comment]
Ready to go?

File: hippiestation/code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -10,6 +18,9 @@
 /obj/item/storage/box/cyber_implants
 	icon = 'icons/obj/storage.dmi'
 
+/obj/item/storage/box/mousetraps
+	illustration = "mousetrap" // fucking tg typoed it
```
Comment (JohnGinnane) [inline hippiestation/code/game/objects/items/storage/boxes.dm:22]: should we just upstream this?

Comment (Amelia0010) [reply hippiestation/code/game/objects/items/storage/boxes.dm:22]: ye

**Comment by Carbonhell** [comment]
Mfw everyone forgot to merge this
good job

**Pull request merged**